src/main/java/kirby/tasks/Task.java

```diff
@@ -0,0 +1,67 @@
+package kirby.tasks;
+
+/**
+ * Task class is an abstract class that is inherited by the specific tasks.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
Great that you have followed the following coding standard - Boolean variables/methods should be named to sound like booleans. Well done!